docker/files/run/run.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+if [ "$ALGORAND_DATA" != "/algod/data" ]; then
```

Comment:
I think there is an alternate algod image which puts the data directory at `/algod`. Someone migrated to the new container and it caused problems because of our logic which looks for all directories at `$ALGORAND_DATA/../*` for setting the tokens.

This is a constant in the script, but docker lets you override variables.

An alternative would be to set it back in this script, but I'd rather make an error so that people don't have unused variables.